docs/readmes/lte/s1ap_tests.md

```diff
@@ -19,23 +19,41 @@ interfere with the test scenario.
 
 Our VM-only tests use 3 Vagrant-managed VMs hosted on the local device (laptop):
 
-- *magma*, i.e. magma-dev or gateway
+- *magma* (a.k.a. magma-dev) or *magma_deb*, both of which act as a gateway, with the difference
+being the way magma is installed on them (see [below](#gateway-vm-setup) for more details)
 - *magma_test*, i.e. s1ap_tester
 - *magma_trfserver*, i.e. an Iperf server to generate uplink/downlink traffic
 
 ## Gateway-only tests
 
 These tests use all 3 VMs listed above. The *magma_test* VM abstracts away the
-UE and eNodeB, the *magma_trfserver* emulates the Internet, while the *magma* VM
+UE and eNodeB, the *magma_trfserver* emulates the Internet, while the *magma*/*magma_deb* VM
 acts as the gateway between *magma_test* and *magma_trfserver*.
 
 ### Gateway VM setup
 
+There are two options for setting up the gateway VM, with the difference being the magma installation
+method: it can either be installed via `make` or from a debian package, the latter being the
+method by which magma is usually deployed. For everyday development, the `make` installation is
+recommended, while the debian installation is useful for testing packages before release.
+
+> **Warning**: These two VMs use the same network configuration, so only one of them should be
+running at any one time.
+
+#### Make installation
+
 Spin up and provision the gateway VM, then make and start its services:
 
 1. From `magma/lte/gateway` on the host machine: `vagrant up magma && vagrant ssh magma`
 1. Now in the gateway VM: `cd $MAGMA_ROOT/lte/gateway && make run`
 
+#### Debian installation
+
+Spin up the *magma_deb* VM. The services should start automatically:
```

Comment:
We might want to remove the `should` here? That would sound more confident.

Reply:
Done :slightly_smiling_face: